green-square-sequence/src/App.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react'
+import 'style.css'
+
+//set how many green squares you want displayed here
+let numOfSquares = 6
+
+export default function App() {
+  let [order, setOrder] = useState([])
+  let squares = [...Array(numOfSquares)]
+
+  const addElement = (num) => {
+    setOrder((order = order.concat(num)))
+  }
+
+  const reverseSequence = async () => {
+    for (let i = 1; i <= numOfSquares; i++) {
+      await delay(500)
+      const newOrder = order.slice(0, order.length - i)
+      setOrder(newOrder)
+    }
+  }
+
+  function delay(time) {
```

Comment:
nit: This function can live outside of `App()` since it doesn't use any local variables.